addon-test-support/ember-qunit/legacy-2-x/module-for-component.js

```diff
@@ -1,6 +1,16 @@
 import { createModule } from './qunit-module';
 import { TestModuleForComponent } from 'ember-test-helpers';
+import { deprecate } from '@ember/application/deprecations';
 
 export default function moduleForComponent(name, description, callbacks) {
   createModule(TestModuleForComponent, name, description, callbacks);
+  deprecate(
+    `The usage "moduleForComponent" is deprecated. Please migrate the "${name}" module to use "setupForRendering".`,
```

Comment:
I believe we want to instruct them to use `setupRenderingTest`, no?

Reply:
Ah you are correct.